lib/templates.ts

```diff
@@ -1513,6 +1514,10 @@ export const updateClosedData = async (
   ]);
   let detailsData: ClosedDetails | null = null;
 
+  if (!isValidISODate(String(closingDate))) {
+    throw new Error("Invalid ISO Date");
```

Comment:
We should add the date that was trying to be passed to the error so we can see it / and fix as needed,